_posts/2019-04-12-Raise_some_horns_Red_Hat_s_MetalKube_aims_to_make_Kubernetes_on_bare_machines_simple.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Raise some horns, Red Hat's MetalKube aims to make Kubernetes on bare machines simple"
+date: 2019-04-12T21:20:00+02:00
+draft: false
+categories: ["hybrid", "cloud", "metal3", "baremetal", "stack", "edge", "openstack", "ironic"]
+author: Pedro Ibáñez Requena
+---
+
+## The Register; Raise some horns: Red Hat's Metal³ aims to make Kubernetes on bare machines simple
+
+[Max Smolaks](https://www.theregister.co.uk/Author/Max-Smolaks) talks in this article about the OpenInfra Days in the UK, 2019: where Metal³ was revealed earlier last week by Steve Hardy, Red Hat's senior principal software engineer. The Open Infrastructure Days in the UK is an event organised by the local Open Infrastructure community and supported by the OpenStack Foundation.
+The Open-source software developers at Red Hat are working on a tool that would simplify deployment and management of Kubernetes clusters on bare-metal servers.
+
+Steve told The reg:
+> "In some situations, you won't want to run a full OpenStack infrastructure-as-a-service layer to provide, potentially, for multiple Kubernetes clusters".
+
+Hardy is a notable contributor to OpenStack, having previously worked on Heat and TripleO projects. He said one of the reasons for choosing Ironic was its active development – and when new features get added to Ironic, the Metal³ team gets them "for free".
+
+> "OpenStack has always been a modular set of projects, and people have always had the opportunity to reuse components for different applications. This is just an example of where we are leveraging one particular component for infrastructure management, just as an alternative to using a full infrastructure API," Hardy said.
+
+Thierry Carrez, veep of engineering at the OpenStack Foundation also told The Reg:
```

Comment:
VP?

Reply:
"veep" is a copy&paste from the original article:

> "I like the fact that the projects end up being reusable on their own, for the functions they bring to the table – this helps us integrate with adjacent communities," Thierry Carrez, veep of engineering at the OpenStack Foundation, told The Reg.

Never read it before, and yes, it's VP.